client/src/App.css

```diff
@@ -1,4 +1,4 @@
-
+@import url('https://fonts.googleapis.com/css2?family=Inconsolata:wght@700&display=swap');
 .logo {
     width: 100px;
 }
```

Comment:
I think it's better to use % instead of px for width

Reply:
This is from the template it is not mine line

Reply:
i only import the font